Sources/MongoSwift/WriteConcern.swift

```diff
@@ -3,6 +3,9 @@ import Foundation
 
 /// A class to represent a MongoDB write concern.
 public struct WriteConcern: Codable {
+    /// Majority WriteConcern.
```

Comment:
This could be more specific, e.g. "A write concern with w: "majority" and the other fields unset."

Reply:
I like the link to the docs, but I think we still need to include information about the other fields. Calling it a "majority write concern" tells you about the `w` field, but it doesn't mention what the `j` or `wtimeout` fields are.

Also, you can replace "see" with `- SeeAlso:` to get nice formatting by our docs generator.

Reply:
As another approach, we could take the path that we did with `ReadPreference` and make these static functions, optionally allowing the user to specify `wtimeout` and `j`.

Reply:
>I like the link to the docs, but I think we still need to include information about the other fields. Calling it a "majority write concern" tells you about the w field, but it doesn't mention what the j or wtimeout fields are.

This portion of my comment still needs to be addressed. I also think the method could have some more info about how the user can optionally specify `j` or `wtimeout`, so as to differentiate it from the static property.